rasa/core/channels/console.py

```diff
@@ -22,6 +22,8 @@
 
 DEFAULT_STREAM_READING_TIMEOUT_IN_SECONDS = 10
 
+CONSOLE_SESSION_USER_ID = str(uuid.uuid4())
```

Comment:
we use the `hex` version of UUID4 elsewhere - it would be good to keep that consistent
```suggestion
CONSOLE_SESSION_USER_ID = uuid.uuid4().hex
```